appshell/appshell_extensions.cpp

```diff
@@ -507,7 +508,35 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
                 error = CopyFile(src, dest);
                 // No additional response args for this function
             }
-        } else if (message_name == "GetDroppedFiles") {
+        } else if (message_name == "SetUpdateParamsAndRunUpdate") {
```

Comment:
Where are you calling this method from in your extension?

Reply:
@mbhavi Can you help me point to the exact location in your PR https://github.com/adobe/brackets/pull/14177. You could click on the line where this is getting called and copy paste the URL from the browser, here.

Reply:
Thanks @mbhavi Got it!